src/components/utils/ControlledAlert/ControlledAlert.tsx

```diff
@@ -74,13 +74,13 @@ const ControlledAlert = React.forwardRef<HTMLDivElement, ControlledAlertProps>(
         >
           <Box>
             {title && (
-              <Flex as="header" align="center" fontSize="large">
+              <Flex as="header" align="center">
                 {icon && <Icon type={icon} mr={2} color={iconColor} size="large" />}
                 <Box
-                  as="h4"
+                  as="h2"
```

Comment:
I'm not sure if `h2` is the right call, but I'm not sure about `h4` either. Wouldn't want to block it on that, so your call :)

Reply:
I wanted to handle alert as individual element, so I used the h2 to specify the importance of the title!